app/src/ui/changes/files-changed-badge.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+
+interface IFilesChangedBadgeProps {
+  readonly numFilesChanged: number
+}
+
+interface IFilesChangedBadgeState {
+  readonly badgeCount: string
+}
+
+function createState(props: IFilesChangedBadgeProps): IFilesChangedBadgeState {
+  if (props.numFilesChanged === 0) {
+    return { badgeCount: '' }
+  }
+
+  /** The number that can be displayed as a specific value */
+  const LargeNumber = 300
+  const badgeCount =
```

Comment:
This can be moved inside of the `render` function that way it's defined near its usage.